viacoin-blockchain-ubuntu/azuredeploy.json

```diff
@@ -190,27 +190,27 @@
         "diagnosticsProfile": {
           "bootDiagnostics": {
             "enabled": "true",
-            "storageUri": "[concat(reference(variables('storageAccountName')).primaryEndpoints.blob, variables('vmStorageAccountContainerName'),'/',variables('OSDiskName'),'.vhd')]"
+            "storageUri": "[concat(refrerence(variables('storageAccountName'),'2015-06-15')).primaryEndpoints.blob,variables('vmStorageAccountContainerName'),'/',variables('OSDiskName'),'.vhd')]"
```

Comment:
Sharp but after correcting it, I still get this error.
To me something like this instead seems correct
`"vhd": {
		"uri": "[concat(reference(concat('Microsoft.Storage/storageAccounts/', parameters('newStorageAccountName')), '2015-06-15').primaryEndpoints.blob, variables('vmStorageAccountContainerName'),'/',variables('OSDiskName'),'.vhd')]"
	}`
But I don't work at microsoft....
If I am correct, please tell me and I will try another git push.

The error I get now is
`"error": "InvalidTemplate : Deployment template validation failed: 'The template resource 'viacoinnode' at line '1' and column '3890' is not valid: Unable to parse template language expression 'concat(reference(variables('storageAccountName'),'2015-06-15')).primaryEndpoints.blob,variables('vmStorageAccountContainerName'),'/',variables('OSDiskName'),'.vhd')': expected token 'EndOfData' and actual 'Comma'. Please see https://aka.ms/arm-template-expressions for usage details..'.\n"
    },`